backend/src/test/java/com/woowacourse/ternoko/service/AuthServiceTest.java

```diff
@@ -36,6 +40,42 @@ public class AuthServiceTest {
     @SpyBean
     private JwtProvider jwtProvider;
 
+    @Test
+    @DisplayName("코치가 본인인지 요청을 보내면 true 를 반환한다.")
+    void checkCrewSameType() {
+        Assertions.assertThatNoException().isThrownBy(() -> authService.checkMemberType(1L, "COACH"));
+    }
+
+    @Test
+    @DisplayName("코치가 아닌데 코치로 요청을 보내면 에러를 반환한다.")
+    void check_coach_Type_false() {
+        final Long crewId = 7L;
+        // when
+        assertThatThrownBy(() -> authService.checkMemberType(crewId, "COACH"))
```

Comment:
테스트 코드 자체로만 보면 7L이 정말 크루의 아이디인지 다른 파일까지 찾아봐야할 것 같아요.
차라리 픽스처를 이용하면 가독성이 더 좋아질 것 같습니다!

```suggestion
        final Long crewId = 7L;
        // when
        assertThatThrownBy(() -> authService.checkMemberType(CREW1.getId(), "COACH"))
```